packages/cli/src/commands/init.ts

```diff
@@ -145,15 +149,26 @@ export const initCommand = async (options: InitCommandOptions) => {
 
   await addConfigurationToPackageJson(resolvedPath, resolvedOptions);
 
-  await printNextSteps(resolvedOptions, authorizedKey);
+  await printNextSteps(resolvedOptions, authorizedKey, nextJsVersion, nextJsDir);
   telemetryClient.init.completed(resolvedOptions);
 };
 
-async function printNextSteps(options: ResolvedOptions, authorizedKey: WhoamiResponse) {
+async function printNextSteps(options: ResolvedOptions, authorizedKey: WhoamiResponse,nextJsVersion: number, nextJsDir:"pages"| "app"  ) {
   const projectUrl = `${options.triggerUrl}/orgs/${authorizedKey.organization.slug}/projects/${authorizedKey.project.slug}`;
 
   logger.success(`✅ Successfully initialized Trigger.dev!`);
 
+
+if (nextJsDir === "pages") {
+  const message = `Detected you are using Next.js Page Router version ${nextJsVersion}, check ${
+    nextJsVersion > 13.1
+      ? "out this article on our docs on how to complete the setup. https://trigger.dev/docs/documentation/quickstarts/nextjs"
+      : "out this article on our docs on how to complete the setup. https://trigger.dev/docs/documentation/quickstarts/nextjs"
```

Comment:
Should these two lines say different things?

Reply:
In which case would it better to just do this:

```
const message = `Detected you are using Next.js Page Router version ${nextJsVersion}, check out this article on our docs on how to complete the setup. https://trigger.dev/docs/documentation/quickstarts/nextjs`
```

Reply:
I'll go with this instead, I couldn't find a way to link to a particular part of a doc on mintlify.